test/e2e/proxy.go

```diff
@@ -242,11 +259,9 @@ func proxyContext(version string) {
 						recordError(fmt.Sprintf("%v: path %v took %v > %v", i, path, d, proxyHTTPCallTimeout))
 					}
 				}(i, path, val)
-				// default QPS is 5
-				time.Sleep(200 * time.Millisecond)
```

Comment:
That leads me to even more questions. The restriction seems to be in place for scale tests. There's no scale here. Disabling the restriction the test finishes in 12 seconds. It spends 99% of it's time downloading the image it uses to run.